pkg/helper/redis_database_version.go

```diff
@@ -0,0 +1,177 @@
+package helper
+
+import (
+	appsv1alpha1 "github.com/3scale/3scale-operator/apis/apps/v1alpha1"
+	"github.com/go-logr/logr"
+
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	redisSystemSecretName            = "system-redis"
+	redisBackendSecretName           = "backend-redis"
```

Comment:
```suggestion
```
Very minor but is there any reason to keep the unused constants?

Reply:
thanks!